populate_dev_data_handler.go

```diff
@@ -86,11 +86,14 @@ func populateDevData(w http.ResponseWriter, r *http.Request) {
 		"safari",
 	} {
 		// TODO(lukebjerring): Move wpt.fyi base URL to constant.
-		jsonURL := "https://wpt.fyi/json?platform=" + browserName
+		jsonURL := "https://wpt.fyi/results?platform=" + browserName
```

Comment:
Is this diff related to this PR?

Reply:
Not really; I'll make this tweak separately.